p7zip.yml

```diff
@@ -0,0 +1,36 @@
+name: p7zip
+no-autogen: true
+make-args:
+    - all2
+    - OPTFLAGS=-O2 -g
+    - DEST_HOME=/app
+    - DEST_BIN=/app/bin
+    - DEST_SHARE=/app/lib/p7zip
+    - DEST_MAN=/app/share/man
+make-install-args:
+    - DEST_HOME=/app
+    - DEST_BIN=/app/bin
+    - DEST_SHARE=/app/lib/p7zip
+    - DEST_MAN=/app/share/man
+sources:
+    - type: git
+      url: https://github.com/jinfeihan57/p7zip
+      commit: 0b5b1b1a866d0e41cb7945e60a32262874e724aa
+      disable-shallow-clone: true
+    - type: shell
+      commands:
+        - git cherry-pick -n -Xours c104127e6a93
+    - type: shell
+      only-arches:
+        - x86_64
```

Comment:
so what happen on an aarch64 ?

Reply:
Er, nothing. This only supports x86-64.

Reply:
then https://github.com/flathub/flathub/wiki/App-Maintenance#only-build-on-x86_64